### PR TITLE
Text color change according to theme

### LINKED ISSUE #4538: [Case Study] HPE: Low contrast of text in dark mode
### Description
This case study has text that is not styled for either light and dark mode - https://layer5.io/resources/cloud-native/hpes-adoption-of-meshery-and-meshmap

### Expected Behavior
Styles applied here are theme-friendly - https://github.com/layer5io/layer5/blob/master/src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/hpe.style.js

Read the CONTRIBUTOR.md for information about theme and light/dark mode treatment.

### Screenshots
Screenshot 2023-07-13 at 8 34 13 AM
Screenshot 2023-07-13 at 8 34 21 AM

### Environment:
- Host OS: macOS
- Browser: Safari

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by HetviSoni**
### **Description**

This PR fixes #4538 

### **Notes for Reviewers**
Earlier the font color did not change according to theme on https://layer5.io/resources/cloud-native/hpes-adoption-of-meshery-and-meshmap

### **Screenshots**
**Before**
imageimage
**After**

https://github.com/layer5io/layer5/assets/76479104/e4076c76-68bc-4a28-a096-77d45dbb5dba

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0494085eb3e38468863d6a4e70a2fa0a00eec0ec at: https://64b0e4450c3c5e0a930742ac--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d31dd5dbd844ea733ce352ea2d12b433a2949ac2 at: https://64b0f256d791e0135ecd43d7--layer5.netlify.app

File: src/collections/resources/Resources.style.js [hunk 1]
```diff
@@ -222,7 +222,14 @@ div.fact-left {
...
 
-
+  /* ==============
+  styles for hpes-adoption-of-meshery-and-meshmap
+  =======================*/
+  
+  .benefits{
+    color: ${props => props.theme.whiteToBlack};
+  }
+
```
Comment (leecalcote) [inline src/collections/resources/Resources.style.js:232]: Please move this to the HPE stylesheet

**Comment by l5io** [comment]
🚀 Preview for commit dbf34fa34b69a433c8118130a9d95f041243ea3c at: https://64b160818137b22bbeb027c2--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit dbf34fa34b69a433c8118130a9d95f041243ea3c at: https://64b1609678765623afd0e682--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit aeb76fdf75eaf374c12d46fd4f76bcc74b4b4ee7 at: https://64b1729c9faf8d37ccf759c6--layer5.netlify.app

File: src/collections/resources/Resources.style.js (hunk 1 above)
Comment (HetviSoni) [reply src/collections/resources/Resources.style.js:225]: @leecalcote I have made the required changes. Please review it.
Note: The div of the 4 benefit points contained inline css, I moved it to the HPE stylesheet for consistency.

**Comment by leecalcote** [comment]
Thank you for including a video recording, @HetviSoni 👍

File: src/collections/resources/Resources.style.js (hunk 1 above)
Comment (leecalcote) [reply src/collections/resources/Resources.style.js:225]: Nice!

**Review by leecalcote — APPROVED** [review]
Looks great, @HetviSoni 💯

File: src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/hpe.style.js [hunk 2]
```diff
@@ -38,11 +38,18 @@ export const HPEintro = styled("div")(({ theme }) => ({
...
   marginBottom: "1rem",
 
   backgroundColor: `${theme.body}`,
 }));
+
+export const HPEbenefits = styled("div")(({ theme }) => ({
+  border: "1px solid #00B39F",
+  padding: "2rem",
+  boxShadow: "rgba(0, 179, 159, 0.2) 0px 10px 25px",
+  color: `${theme.whiteToBlack}`,
```
Comment (leecalcote) [inline src/collections/resources/articles/hpes-adoption-of-meshery-and-meshmap/hpe.style.js:54]: Good job using this.

**Pull request merged**